README.md

```diff
@@ -56,13 +56,13 @@ System.out.println(response);
 #### PingServer
 ```bash
 mvn package
-java -cp tchannel-example/target/tchannel-example-1.0-SNAPSHOT.jar com.uber.tchannel.ping.PingServer -p 8888
+java -cp tchannel-example/target/tchannel-example-0.1.0-SNAPSHOT.jar com.uber.tchannel.ping.PingServer -p 8888
```

Comment:
Are you talking bout the built projects jars? I can setup a maven shade execution to just create a jar for each of our examples (server and client) which will make running and documenting them easier